app/app/src/main/java/com/globant/calculator/android/mvp/presenter/CalculatorPresenter.java

```diff
@@ -23,32 +28,56 @@ public void onClearButtonPress() {
         view.showResult(ZERO_INT);
     }
 
-    public void onOperatorPressed(String symbol){
+    public void onOperatorPressed(String symbol) {
         model.setOperator(symbol);
         view.showOperationPressed(symbol);
     }
 
-    public void onEqualsButtonPressed(String symbol) {
-        // will be added when implements the equals button
+    public void onEqualsButtonPressed() {
+        if (model.getOperator().equals(EMPTY_STRING)) {
```

Comment:
you can use model.getOperator().isEmpty()

Reply:
switched, thanks